build.sbt

```diff
@@ -79,7 +79,7 @@ val sharedSettings = scalariformSettings ++  Seq(
 
   scalacOptions ++= {
     if (scalaVersion.value startsWith "2.12")
-      Seq("-opt:l:classpath")
+      Seq("-opt-inline-from:com.twitter.algebird.**")
```

Comment:
make sure we are only inlining from the local project. I *think* this is the correct invocation.

Reply:
you **also** need the `-opt:l:inline` flag, `-opt-inline-from` alone doesn't enable the optimizer, it just configures it.